internal/util/util.go

```diff
@@ -170,3 +177,16 @@ func CreateRandomJitterDuration(max int64, duration time.Duration) time.Duration
 	jitteredDuration := duration * time.Duration(n)
 	return jitteredDuration
 }
+
+func LabelsMatchLabelSelector(labels map[string]string, labelSelector map[string]string) bool {
```

Comment:
This is expected but I think it would be good to document empty label selector is a match

Reply:
I don't see it this way... I think it's a bug.  A fleet with no label selector should match nothing, not everything.  I changed the code to match this behavior.

Reply:
OK I guess we should decide if we are going to maintain k8s parity or not.

https://github.com/kubernetes/kubernetes/blob/54f9807e1e84981b2053f4daf779f5ed19962144/staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/types_swagger_doc_generated.go#L166

Reply:
You're right, this is more consistent.  Changed.